grid/social-media/posts/maps/linkedin.suma

```diff
@@ -0,0 +1,107 @@
+profile = "social-media/posts@1.0"
+provider = "linkedin"
+
+// Uses LinkedIn's UGC Posts API: https://docs.microsoft.com/en-us/linkedin/marketing/integrations/community-management/shares/ugc-post-api#find-ugc-posts-by-authors
+// There's a posts API beta but currently it seems broken for even basic retrieval: https://docs.microsoft.com/en-us/linkedin/marketing/integrations/community-management/shares/posts-api-beta
+
+"""
+Retrieve posts from LinkedIn profile
+Requires `r_organization_social` scope for reading data from organization's profile.
+"""
+map GetProfilePosts {
+  count = 50
+  // https://stackoverflow.com/a/60450805/240963
+
+  projection = '(paging,elements*(id,created,specificContent(com.linkedin.ugc.ShareContent(shareMediaCategory,shareCommentary,media(*(media~:playableStreams,originalUrl,description,title,thumbnails))))))'
+
+  requestPath = input.page || `v2/ugcPosts?q=authors&sortBy=LAST_MODIFIED&authors=List(${encodeURIComponent(input.profileId)})&count=${count}&projection=${projection}`
+  // FIXME: I can't use just bare {requestPath} here since SDK assumes it's an absolute URL...
+  http GET "/{requestPath}" {
+    request "application/json" {
+      headers {
+        "X-Restli-Protocol-Version" = "2.0.0",
+        "Authorization" = `Bearer ${parameters.accessToken}`,
+      }
+    }
+  
+    response 200 "application/json" {
+      result = body.paging.links.reduce((acc, current) => {
+        const {rel} = current;
+        // Hack for "relativizing" URL
+        const href = current.href.substring(1);
```

Comment:
Just an idea: we could encode page references to be safe to use in query parameters.

Reply:
I have resorted to returning just the `start` parameter.